src/pybind/mgr/rook/rook_cluster.py

```diff
@@ -1034,18 +1073,23 @@ def add_osds(self, drive_group, matching_hosts):
             self.creator = LSOCreator(inventory, self.coreV1_api, self.storage_class)    
```

Comment:
this is old code, but no reason for `creator` to be part of `self`.. just make it a local variable